pyproject.toml

```diff
@@ -59,6 +59,9 @@ source = ["src/parver", ".tox/*/lib/python*/site-packages/parver"]
 precision = 1
 exclude_lines = ["pragma: no cover", "pass"]
 
+[tool.pytest.ini_options]
+pythonpath = "src"
```

Comment:
This isn't correct. The `src` directory exists so that the package is *not* on `PYTHONPATH` by default. This is so that the package must be installed before running tests.